backend/src/server.ts

```diff
@@ -1,9 +1,12 @@
 import express from 'express';
 import packagesRouter from './routes/packages';
 import packageRouter from './routes/package';
+import searchPackagesRouter from './routes/search_packages';
 import cors from 'cors';
 const app = express();
 
+require('dotenv').config();
```

Comment:
Should we require dotenv or should we just manually export these variables?

Reply:
@rakshittonmoy

Reply:
To my understanding dotenv does the same as manually exporting but allows use of a file

May actually be easier instead of requiring manual exports every time, just add to gitignore as well

Reply:
Yes, it is already in the gitignore file.

Reply:
Sounds good to me then!